snippets/graph_neighbourhood.py

```diff
@@ -0,0 +1,68 @@
+""""""
+
+from itertools import starmap
+from operator import le, lt, and_
+from pathlib import Path
+from os import path
+from sys import argv, stdout
+from argparse import ArgumentParser
+
+from data import load_graph
+
+
+def neighbourhood(coord: tuple[int, int], borders) -> None:
+    n, s, w, e, nw, ne, sw, se = [
```

Comment:
Please rename the variables to give them a more meaningful context. For e.g., 'n' could be renamed to north, nw could be renamed to northwest assuming that these variables represent the directions.